apps/web/src/app/globals.css

```
@@ -476,6 +476,197 @@
+    }
+    100% {
+      background-color: var(--color-surface);
+    }
+  }
+  @keyframes om-flag-fill {
+    0% {
+      fill: transparent;
+    }
+    12%,
+    88% {
+      fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)